opentech/apply/activity/messaging.py

```diff
@@ -365,24 +437,75 @@ def send_message(self, message, submission, subject, recipient, **kwargs):
             return 'Error: ' + str(e)
 
 
+class DjangoMessagesAdapter(AdapterBase):
+    adapter_type = 'Django'
+    always_send = True
+
+    messages = {
+        MESSAGES.BATCH_REVIEWERS_UPDATED: 'batch_reviewers_updated',
+    }
+
+    def reviewers_updated(self, added=list(), removed=list(), **kwargs):
+        message = ['Reviewers updated.']
+        if added:
+            message.append('Added:')
+            message.append()
+
+        if removed:
+            message.append('Removed:')
+            message.append(', '.join([str(user) for user in removed]) + '.')
+
+        return ' '.join(message)
```

Comment:
Is this method used somewhere?

Reply:
no, i just copy pasted it and forgot to remove it...